src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,22 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
+    public static final String BASE_PACKAGE_NAME = "mate.academy";
```

Comment:
There is a missing semicolon at the end of the line declaring `BASE_PACKAGE_NAME`. This will cause a syntax error. Please add a semicolon at the end of this line.